core/server/common/src/main/java/alluxio/underfs/AbstractUfsManager.java

```diff
@@ -133,27 +134,30 @@ private UnderFileSystem getOrAdd(AlluxioURI ufsUri, UnderFileSystemConfiguration
   }
 
   @Override
-  public UfsInfo addMount(long mountId, AlluxioURI ufsUri,
-      UnderFileSystemConfiguration ufsConf) {
+  public void addMount(long mountId, final AlluxioURI ufsUri,
```

Comment:
Why do we change this to `void`, it seems like in most cases we directly try to `get` afterward.